packages/react/src/component-tests/IcSelect/IcSelect.cy.tsx

```diff
@@ -0,0 +1,2627 @@
+/// <reference types="Cypress" />
+/* eslint-disable sonarjs/no-duplicate-string */
+/* eslint-disable cypress/no-unnecessary-waiting */
+
```

Comment:
Is the plan to revisit the eslint disabled errors later?

Reply:
I could fix them, it'll be a slog
![Screenshot 2023-12-15 at 08 59 32](https://github.com/mi6/ic-ui-kit/assets/111292477/c79cd143-a012-4da7-b227-6faccedf479d)